src/storage/hub.ts

```
@@ -72,20 +74,7 @@ export async function deleteFromGaiaHub(
     }
   )
   if (!response.ok) {
-    let responseMsg = ''
-    try {
-      responseMsg = await response.text()
-    } catch (error) {
-      Logger.debug(`Error getting bad http response text: ${error}`)
-    }
-    const errorMsg = 'Error deleting file from Gaia hub: '
-      + `${response.status} ${response.statusText}: ${responseMsg}`
-    Logger.error(errorMsg)
-    if (response.status === 404) {
-      throw new FileNotFound(errorMsg)
-    } else {
-      throw new Error(errorMsg)
-    }
+    throw getBlockstackErrorFromResponse(response, 'Error deleting file from Gaia hub.')
```

I think this needs an `await`

---

That may explain some of the failing tests, a few `UnhandledPromiseRejectionWarning: Unhandled promise rejection` warnings in the logs.

---

Looking closer at the failures I found that there was 1 failure caused by my change to `getFile` behavior (missing files throw `DoesNotExist` error instead of returning null), and the rest were failing because that one failed. Once I fixed this test the rest all started passing 😀.